protocol/bc/outputid.go

```diff
@@ -0,0 +1,44 @@
+package bc
+
+import (
+	"io"
+
+	"chain/crypto/sha3pool"
+	"chain/encoding/blockchain"
+)
+
+// OutputID identifies previous transaction output in transaction inputs.
+type OutputID struct{ Hash }
```

Comment:
Why `struct{Hash}` and not simply `Hash`, as we do for e.g. `AssetID` and other types?